docker/docker.go

```diff
@@ -605,11 +689,18 @@ func (d *ImageManager) RunInContainer(opts RunInContainerOpts) (exitCode int, co
 	} else {
 		exitCode = -1
 	}
+
+	// Stream files out of the container
+	streamErr := streamOutFiles()
+	if streamErr != nil {
+		return exitCode, container, fmt.Errorf("Error running in container: %s. Error streaming data out of container: %s", container.ID, streamErr)
+	}
 	closeFiles()
+
```

Comment:
This function has named return values: `(exitCode int, container *dockerclient.Container, err error)`. Can we get rid of the names; they don't seem to be used?

Reply:
`exitCode` is used